R/CreatePackageReport.R

```diff
@@ -33,59 +35,54 @@ CreatePackageReport <- function(packageName
         log_fatal(msg)
     }
     
-    log_info(paste0("Creating package report for package "
-                   , packageName
-                   , " with reporters:\n\n"
-                   , paste(unlist(lapply(packageReporters, function(x) class(x)[1]))
-                           , collapse = "\n")))
+    # Fit reporters with package name and (maybe) package path
+    reporters <- lapply(
+        packageReporters
+        , function(reporter, packageName, packagePath){
+            reporter$set_package(packageName, packagePath)
+            reporter$calculate_all_metrics()
+            return(reporter)
+        }
+        , packageName = packageName
+        , packagePath = packagePath
+    )
 
-    # Make them plots
-    for (reporter in packageReporters){
-        log_info(paste("Running Package Reporter", class(reporter)[1]))
-        reporter$set_package(packageName, packagePath)
-        
-        reporter$calculate_all_metrics()
-        # TODO: replace plot_network() with render_report() which is then rendered into a parent report.
-        reporter$plot_network()
-        log_info(paste("Done Package Reporter",class(reporter)[1]))
-    }
-    
-    RenderPackageReport(
+    .RenderPackageReport(
```

Comment:
Good Call.  We also need to discuss what functions are displayed in pkgdown website.